src/import-mongo.js

```diff
@@ -0,0 +1,63 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import CouchBulkImporter from 'couchbulkimporter';
+import mongo from 'mongodb';
+import BulkBadger from 'bulkbadger';
+import { getUrlFromCluster } from './utils';
+
+export function cli (cluster, database,  mongourl, collection) {
+  if (!cluster || !database || !mongourl || !collection) {
+    const msg = [
+      'Usage:',
+      '',
+      'nmo import-mongo [clustername] [database] [MongoDB-url] [collection]',
+      'nmo import-mongo [url] [database] [MongoDB-url] [collection]'
+    ].join('\n');
+    const err = new Error(msg);
+    err.type = 'EUSAGE';
+
+    throw err;
+  }
+
+  return importmongo(cluster, database, mongourl, collection);
+}
+
+
+export default importmongo;
+function importmongo (cluster, database, mongourl, collection) {
+  return new Promise((resolve, reject) => {
+    mongo.connect(mongourl, function (err, db) {
+      if (err) {
+        reject(err);
+        return;
+      }
+
+      const clusterUrl = getUrlFromCluster(cluster);
+      const col = db.collection(collection);
+      col.find({}, {})
+        .on('error', function (err) {
+          err.message = 'Error fetching - ' + err.message;
+          reject(err);
+        })
+        .pipe(new BulkBadger())
+        .on('error', function (err) {
+          err.message = 'Error converting - ' + err.message;
```

Comment:
db.close(); missing in the upper two cases, but maybe not so important as the command finishes anyway with process.exit afterwards